itou/static/js/sliding_tabs.js

```diff
@@ -1,24 +1,28 @@
-// Only for <ul.s-tabs-01__nav.nav.nav-tabs> how has data-it-sliding-tabs="true" attribute
-let slidingTabs = document.querySelector("[data-it-sliding-tabs=true]");
+"use strict";
 
-if (slidingTabs) {
-  let slidingTabsStartIndex = 0;
+htmx.onLoad(function (target) {
+  // Only for <ul.s-tabs-01__nav.nav.nav-tabs> how has data-it-sliding-tabs="true" attribute
```

Comment:
Je crois qu'il y a une une coquille `how` <-> `now`. 

Mais surtout je pense que le commentaire n'est plus d'actualité : 
dans siaes_search_tabs.html je vois un `s-tabs-01__nav nav nav-tabs` sans `data-it-sliding-tabs="true"`

Reply:
Yes, le commentaire n'est plus vraiment utile.
Pour le `s-tabs-01__nav nav nav-tabs` sans `data-it-sliding-tabs="true"`, c'était voulu, mais ça changera surement ;)